pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -56,6 +59,36 @@ func CreateBlobAccessObjectsFromConfig(configurationFile string) (blobstore.Blob
 	return contentAddressableStorage, actionCache, nil
 }
 
+func newClientTLSFromFiles(serverCaFile string, clientCertFile string, clientKeyFile string) (grpc_credentials.TransportCredentials, error) {
```

Comment:
Probably more elegant to pass in a pointer to the proto message here. That way the switch below doesn't need to be refactored in case more TLS options are added.